lib/resources/DNSRecords.js

```diff
@@ -44,6 +44,22 @@ module.exports = auto(
       path: ':id',
     }),
 
+    /**
+     * bindConfig allows for retrieving all DNS Records for a zone in bind configuration format.
+     *
+     * @function bindConfig
+     * @memberof DNSRecords
+     * @instance
+     * @async
+     * @param {string} zone_id - The zone ID
+     * @returns {Promise<Object>} The DNS browser response object.
+     */
+    bindConfig: method({
```

Comment:
On the WIP branch, the `import` functionality was added as [`imoport`](https://github.com/cloudflare/node-cloudflare/commit/b6a097c58995dd087da4a4b7fddc4b43f340783a#diff-cb8c622321f605000798bcdacbece2e8R55). To mirror, this should probably be `export`.